src/frontend/src/scheduler/plan_fragmenter.rs

```diff
@@ -391,9 +392,9 @@ impl BatchPlanFragmenter {
                     builder.root = Some(Arc::new(execution_plan_node));
                 }
                 // Check out the comments for `has_table_scan` in `QueryStage`.
-                if let Some(scan_node) = node.as_batch_seq_scan() {
-                    let table_desc = scan_node.logical().table_desc();
-
+                let scan_node: Option<&BatchSeqScan> = node.as_batch_seq_scan();
+                if let Some(scan_node) = scan_node {
+                    // TODO: handle multiple table scan inside a stage
```

Comment:
🤔 To achieve this, is it better to put vnodes in fragment level instead of `BatchSeqScan` node?

Reply:
No, one stage may have multi table scan, think about join push down.

Reply:
This problem is not introduced and won't be handled in this PR

Reply:
Just added a TODO